### DISCUSSION TITLE
Choosing date and time

### DISCUSSION THREAD

**Discussion by felixwiemuth** [felixwiemuth/SimpleReminder#16] [Ideas and feedback]
One of the main goals of SimpleReminder is efficiency in creating a reminder.
One can think of many ways of how the app lets one choose date and time for a reminder.
What would your favourite way(s) be?

**Comment by FaqatZubair1** [comment]
When we choose a time in 12 hour format, we are unable to select the "a.m or p.m" for it, like if I want to keep a reminder at 5 p.m, there is no way to keep it without using the 24hr format.

**Reply by felixwiemuth** [reply]
I guess you mean the same feature request as #33 ?

**Reply by FaqatZubair1** [reply]
Yes, it is somewhat the same issue.

**Reply by felixwiemuth** [reply]
When that is implemented, you shouldn't have any issues with 12-hour format.
